check_duplicates.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
```

Comment:
This file should be in `tools/scripts`, and probably have a more obvious filename (`checkDuplicateFilenames`?).